consensus/accept.go

```diff
@@ -108,9 +90,23 @@ func (s *State) validHeader(b Block) (err error) {
 		return
 	}
 
-	// Check that the block is the correct size.
-	if len(encoding.Marshal(b)) > BlockSizeLimit {
-		return LargeBlockErr
+	return
+}
+
+// addBlockToTree takes a block and a parent node, and adds a child node to the
+// parent containing the block. No validation is done.
+func (s *State) addBlockToTree(b Block) (err error) {
```

Comment:
What is the rationale for moving this to accept.go instead of keeping it in blocknode.go?

I guess it makes more sense in accept.go than in block.go, but renaming from block.go to accept.go weakens the connection between the things like validHeader and ErrLargeBlock.